LeaderboardBackend.Test/Fixtures/PostgresDatabaseFixture.cs

```diff
@@ -0,0 +1,106 @@
+using System;
+using System.Threading.Tasks;
+using DotNet.Testcontainers.Builders;
+using DotNet.Testcontainers.Configurations;
+using DotNet.Testcontainers.Containers;
+using LeaderboardBackend.Test.Fixtures;
+using Npgsql;
+using NUnit.Framework;
+
+// Fixtures apply to all tests in its namespace
+// It has no namespace on purpose, so that the fixture applies to all tests in this assembly
+
+[SetUpFixture] // https://docs.nunit.org/articles/nunit/writing-tests/attributes/setupfixture.html
+internal class PostgresDatabaseFixture
+{
+	public static TestcontainerDatabase? PostgresContainer { get; private set; }
+	public static bool HasCreatedTemplate { get; private set; } = false;
+
+	private static string TemplateDatabase => PostgresContainer?.Database + "_template";
+
+	[OneTimeSetUp]
+	public static async Task OneTimeSetup()
+	{
+		if (TestConfig.DatabaseBackend != DatabaseBackend.TestContainer)
+		{
+			return;
+		}
+
+		PostgresContainer = new ContainerBuilder<PostgreSqlTestcontainer>()
```

Comment:
Github Actions are warning that this constructor's obsolete. Mind fixing this accordingly?